docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,143 @@
+Certora Prover Conf File Format
+===============================
+
+Conf files are an alternative way for setting arguments for the
+ `certoraRun` tool. In terms of functionality 
+using conf files is identical to the use of the [CLI Options](options.md). Instead of calling `certoraRun` 
+with a list of shell flags, some or all the flags can be stored in a JSON file 
+(to be more precise the format is [JSON5](https://json5.org/)):
+
+```
+certoraRun my_params.conf
+```
+
+* Conf files must use the `.conf` suffix
+
+
+
+How CLI Options are mapped to JSON
+----------------------------------
+
+Command-line arguments are stored as key-value pairs in the conf file. 
+The keys are the names of the parameters (with the leading `--` removed). 
+For example,
+```
+certoraRun --verify Example:example.spec
+```
+is equivalent to running with the following conf file:
+
+```
+{ "verify": "Example:example.spec" }
+```
+The values in the map depend on the type of arguments:
+
+* boolean flags that take no arguments (such as {ref}`--send_only`), 
+the value should be `true`. For example,
```

Comment:
Incomplete sentence.  Perhaps "Boolean flags that take no arguments (such as `--send_only`) should have a value of `true`."

Reply:
done